### ISSUE TITLE
Warning on 64bit system - Integer conversion

### ISSUE THREAD

**Issue by woldendans** [symisc/unqlite#23]
Hello,

On a windows 64 bit system compiling in C for 64bit architecture, I get:

```
unqlite.c(29339) : warning C4244: '=' : conversion from '__int64' to 'int', possible loss of data
unqlite.c(29351) : warning C4244: '=' : conversion from '__int64' to 'int', possible loss of data
unqlite.c(29460) : warning C4244: '=' : conversion from '__int64' to 'int', possible loss of data
```

This makes sense. From what I see, you do pointer math and store the result in an int. In 64bit, pointers are 64bit and int is usually 32bit (on Windows at least). The loss of data could be real, even if really unlikely.

**Comment by symisc** [comment]
Thx for the report. Will check on that.

**Comment by symisc** [comment]
Fixed in 1.1.8
